bugwarrior/services/github.py

```diff
@@ -352,9 +360,8 @@ def issues(self):
 
         issues = {}
         if self.involved_issues:
-            issues.update(
-                self.get_involved_issues(self.username)
-            )
+            involved = self.get_involved_issues(self.username)
+            issues.update(filter(self.filter_issues, involved.items()))
```

Comment:
It appears that the reason the key is important here is that an issue can appear for multiple reasons (assignment, in a repo I care about, mentioned, etc.). The key is the URL of the issue so that any duplicates get dropped this way.